beginner_source/nlp/advanced_tutorial.py

```diff
@@ -217,7 +217,7 @@ def _forward_alg(self, feats):
     def _get_lstm_features(self, sentence):
         self.hidden = self.init_hidden()
         embeds = self.word_embeds(sentence).view(len(sentence), 1, -1)
-        lstm_out, self.hidden = self.lstm(embeds)
+        lstm_out, self.hidden = self.lstm(embeds, self.hidden)
```

Comment:
Yes, this was formerly required but PyTorch LSTMs have had automatic initialization with zeros since February or so.